### PR TITLE
Tool Switcher

### PR DESCRIPTION
**Pull request by monster860**
https://streamable.com/50jt6

:cl: monster860
rscadd: Adds tool switcher. Load it up with tools, and program a sequence of tools, then use it in your hand to play through that sequence
/:cl:

### CODE REVIEW

**Comment by Ratvar** [comment]
Does it work on surgeon tools?
Why not make it work on *anything*?

**Comment by Moja10** [comment]
Looks broken beyond words. Should be traitor nukeop only imo

**Comment by ghost** [comment]
Not faster than any of the upgraded tools, but it does eliminate the need for a toolbelt.

It's basically an arm-mounted surgical toolkit implant, but disarmable, and without the increased movespeed.

Give it a mineral req and it should be fine to shove over 'experimental tools'

**Comment by monster860** [comment]
It can do surgery but the circular saw doesn't fit in side so it's not that useful for that.

**Comment by jcatjmeow** [comment]
"program a sequence of tools"
You guys are forgetting this part
10/10 i love this more than my son

File: code/modules/research/designs/tool_designs.dm [hunk 1]
```diff
@@ -33,6 +33,16 @@
 	category = list("Tool Designs")
 	departmental_flags = DEPARTMENTAL_FLAG_SCIENCE | DEPARTMENTAL_FLAG_ENGINEERING
 
+/datum/design/tool_switcher
```
Comment (ThatLing) [inline code/modules/research/designs/tool_designs.dm]: yogs start

**Review by ThatLing — CHANGES_REQUESTED** [review]

File: code/modules/research/designs/tool_designs.dm (hunk 1 above)
Comment (monster860) [reply code/modules/research/designs/tool_designs.dm]: there fixed

**Pull request merged**